src/riscv-legacy-integration.adoc

```diff
@@ -250,13 +250,33 @@ cause illegal instruction exceptions
 addresses (xref:csr-numbers-section[xrefstyle=short]) cause illegal
 instruction exceptions
 * All allowed instructions execute as if the CHERI execution mode is Legacy.
-The CME bits in <<mseccfg>>, <<menvcfg>>, and <<senvcfg>> have no effect whilst
+The CME bits in <<menvcfg>> and <<senvcfg>> are zero whilst
```

Comment:
This could use a bit of clarification. 
* If senvcfg.CRE=0 then senvcfg.CME=0
* If menvcfg.CRE=0 then menvcfg.CME=0 && senvcfg.CRE=0 && senvcfg.CME=0

Reply:
that's in the table already